.travis.yml

```diff
@@ -20,14 +20,15 @@ env:
   - REQUIREMENTS=devel
 
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
+   - "3.6"
```

Comment:
Major: Indentation is wrong (one extra space) for the three lines of python versions